src/OpenLoco/src/Objects/TownNamesObject.h

```diff
@@ -13,17 +13,17 @@ namespace OpenLoco
 #pragma pack(push, 1)
     struct TownNamesObject
     {
-        struct Unk
+        struct Category
         {
             uint8_t count;
-            uint8_t fill;
-            uint16_t offset;
+            uint8_t bias;
+            StringId offset;
```

Comment:
this doesn't look like its a stringid.

Reply:
Ahh, you're right, that was a wrong assumption on my part earlier on.